### ISSUE TITLE
Issues processing background property

### ISSUE THREAD

**Issue by azza85** [jakubpawlowicz/clean-css#332]
This issue was originally posted as a grunt-contrib-cssmin issue.
https://github.com/gruntjs/grunt-contrib-cssmin/issues/117

If I have the below css

```
.test {
  background: none repeat scroll 0 0 #000;
}
```

After I grunt it I get the below result.

```
.test{background:none}
```

Which means the background colour is no longer #000.

in my Gruntfile
I have the below

```
    cssmin: {
      combine: {
        files: {
          'css/main-min.css': ['css/main.css']
        }
      }
    }
```

It seems to be removing everything after none?

Let me know if any further information is required here?
I have the below versions installed.

├── grunt@0.4.5
├── grunt-contrib-cssmin@0.10.0

**Comment by ralphsmith80** [comment]
Since this is closed I'm guessing this suppose to be working now. I'm still seeing it using [grunt-contrib-cssmin](https://github.com/gruntjs/grunt-contrib-cssmin) v0.11.0. As noted by @legomushroom setting the --skip-advanced options works.

**Comment by jakubpawlowicz** [comment]
@ralphsmith80 could you post an example?

**Comment by jakubpawlowicz** [comment]
You may also prefer `--skip-shorthand-compacting` as it skips less optimizations.

**Comment by ralphsmith80** [comment]
Here's the a sample css snippet that breaks.
``` css
.app-logo {
  background: url('img/glad/admin-logo-header.png') no-repeat center;
  background-color: none;
}
```

Here's the optimized output I'm getting. 
```css
.app-logo{background:url(img/glad/admin-logo-header.png) center no-repeat none}
```

Notice `none` is getting added as attribute on `background` which breaks when a url is defined. I suspect it may only be a problem when using a url with the background.

**Comment by jakubpawlowicz** [comment]
@ralphsmith80 it's a slightly different behavior, but a bug nonetheless. I've just created #447. Fix is coming!

**Comment by jakubpawlowicz** [comment]
@ralphsmith80 Fixed in 3.0.8.

**Comment by ralphsmith80** [comment]
Thanks!
